README.md

```diff
@@ -31,26 +31,43 @@ setup and preferences.
   be installed via `brew install git-lfs`.
 * **Windows users**. Git LFS is included in the distribution of 
   [Git for Windows](https://gitforwindows.org/). Alternatively, you can 
-  install a recent version of Git LFS from the Chocolatey package manager.
-
-In addition, [binary packages](https://github.com/git-lfs/git-lfs/releases) are
-available for Linux, macOS, Windows, and FreeBSD. This repository can also be
+  install a recent version of Git LFS from the [Chocolatey](https://chocolatey.org/) package manager.
+* **Binary packages**. In addition, [binary packages](https://github.com/git-lfs/git-lfs/releases) are
+available for Linux, macOS, Windows, and FreeBSD.
+* **Building from source**. [This repository](https://github.com/git-lfs/git-lfs.git) can also be
 built from source using the latest version of [Go](https://golang.org), and the
 available instructions in our
 [Wiki](https://github.com/git-lfs/git-lfs/wiki/Installation#source).
 
-### Usage
+### Installing
+
+#### From binary
+
+The [binary packages](https://github.com/git-lfs/git-lfs/releases) include a script which will:
 
-Git LFS requires a global installation once per-machine. This can be done by
+- Install Git LFS binaries onto the system `$PATH`
+- Run `git lfs install` to
```

Comment:
It's not "global"; it's per-user.  When you say "global" it makes me want to run sudo.

Reply:
Its "global" for the user?

Reply:
Global as in worldwide!   Anyway, this was the original wording. It means git global as opposed to per repo. `git config --global -l | fgrep lfs`